application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -72,15 +72,15 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
     public void onAutoComplete(CommandAutoCompleteInteractionEvent event) {
         AutoCompleteQuery focusedOption = event.getFocusedOption();
 
-        if (focusedOption.getName().equals(ID_OPTION)) {
-            return;
+        if (!focusedOption.getName().equals(ID_OPTION)) {
+            throw new IllegalArgumentException("Unexpected option");
```

Comment:
the exception would be a lot more helpful if u actually add what it was to the message. `"Unexpected option, was: " + ...`